lldb/source/Core/Address.cpp

```diff
@@ -680,21 +729,28 @@ bool Address::Dump(Stream *s, ExecutionContextScope *exe_scope, DumpStyle style,
             if (show_stop_context) {
               // We have a function or a symbol from the same sections as this
               // address.
-              sc.DumpStopContext(s, exe_scope, *this, show_fullpaths,
-                                 show_module, show_inlined_frames,
-                                 show_function_arguments, show_function_name);
+              // Using the same logic, hecking if searched symbol passed to this function or if it using the defualt nullptr
+              if(name)
+                sc.DumpStopContext(s, exe_scope, *this, show_fullpaths,
+                                  show_module, show_inlined_frames,
+                                  show_function_arguments, show_function_name, name);
+              else
+                sc.DumpStopContext(s, exe_scope, *this, show_fullpaths,
+                                  show_module, show_inlined_frames,
```

Comment:
Again here we can do one call that passes on `name` always. If it's nullptr, that's fine just pass it on and no colouring is done.